snmp/datadog_checks/snmp/data/profiles/arista.yaml

```diff
@@ -11,7 +22,57 @@ extends:
 device:
   vendor: "arista"
 
-sysobjectid: 1.3.6.1.4.1.30065.*
+# The OID `1.3.6.1.4.1.30065.1`/`aristaProducts` is the root object identifier from
+# which sysObjectID values are assigned. Values are defined in ARISTA-PRODUCTS-MIB.
+sysobjectid: 1.3.6.1.4.1.30065.1.*
+
+# Note related to use of index `1` of entPhysicalTable columns:
+# Based on snmpwalks of Arista, the main/chassis hardware have the index `1`
+# Librennms is also using index `1`: https://github.com/librenms/librenms/blob/1ac60e3b1d90616119f3c4adc28213e3c35c2477/includes/definitions/discovery/arista_eos.yaml#L4
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "arista"
+      serial_number:
+        symbol:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.11.1  # entPhysicalSerialNum OID at index `1`
+          name: entPhysicalSerialNum
+          # Examples: `HSH16195058`
+      version:
+        symbol:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.8.1  # entPhysicalHardwareRev OID at index `1`
+          name: entPhysicalHardwareRev
+          # Examples: `12.00`
+      product_name:
+        symbol:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.2.1  # entPhysicalDescr OID at index `1`
+          name: entPhysicalDescr
+          # Examples: `DCS-7504 Chassis`
+      model:
+        symbols:
+          - OID: 1.3.6.1.2.1.47.1.1.1.1.13.1  # entPhysicalModelName OID at index `1`
+            name: entPhysicalModelName
+            # Examples: `Arista Networks CCS-720XP-48Y6`
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Based on the recording example should be `CCS-720XP-48Y6`

Reply:
👍  fixed here https://github.com/DataDog/integrations-core/pull/10944/commits/84263b668da21e042498c7d279ec1f269c00bb8a